accounting/accounting_cli.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import sqlite3
+import argparse
+import time
+import sys
+
+import pandas as pd
+
+from accounting import accounting_cli_functions as aclif
+
+def msg(name=None):
+    return """
+    Description: accounting_cla will translate command line arguments into
+    SQLite instructions for the Flux Accounting Database.
+
+    Usage: python cla_hostlist.py [OPTION]...
+
+    -h, --help                              Display this message.
+    -u, --user USER                         View account information for a specific user.
+    -a, --add_user FIELDS                   Add a user to the accounting database.
+
+        FIELDS=user_name admin_level account parent_acct shares max_jobs max_wall_pj
+
+    -d, --delete_user USER                  Remove a user from the accounting database.
+    -e, --edit_value USER FIELD NEW_VALUE   Edit an existing user's value in the accounting database
+    """
+
+
+def parse_args(args):
+    parser = argparse.ArgumentParser(usage=msg())
+
+    parser.add_argument(
+        "-u", "--user", dest="view_user", help="view a user's account information"
+    )
+    parser.add_argument(
+        "-a",
+        "--add-user",
+        dest="add_user",
+        nargs="*",
```

Comment:
I ended up going with `metavars`. The output help message seems pretty useful to me now, but what do you think? Does a sub parser still make more sense to implement? 

```
usage: flux-accounting [-h] [-u USER_NAME]
                       [-a USER_NAME ADMIN_LEVEL ACCOUNT PARENT_ACCT SHARES MAX_JOBS MAX_WALL_PJ]
                       [-d USER_NAME] [-e USER_NAME FIELD NEW VALUE]
```

Reply:
I think someone commented below that a subcommand might be good b/c that'll match the rest of flux's command line usage.  Not a requirement, but something to consider.

Reply:
Oh oh! I'm sorry. I think I was misunderstanding. Instead of running something like `flux-accounting -u fluxuser`, the suggestion was to use subcommands (like `flux mini submit`):

```
$ flux-accounting view-user fluxuser

<output>
```